app/views/public_pages/home.html.erb

```diff
@@ -18,14 +18,14 @@
           </strong>
         </p>
 
-        <% if Rails.env.production? %>
-          <%= link_to vita_providers_path, id: "firstCta", class: "button button--primary" do %>
-            Find a location
-          <% end %>
-        <% else %>
+        <% if enable_online_intake? %>
           <%= link_to question_path(QuestionNavigation.first), id: "firstCta", class: "button button--primary" do %>
             Get started
           <% end %>
+        <% else %>
```

Comment:
i think we decided that the cta button at the bottom of the page should behave in the same way as this one

Reply:
missed that, thanks!